redfish-core/lib/pcie.hpp

```diff
@@ -456,6 +457,20 @@ inline void requestRoutesSystemPCIeDevice(App& app)
                             }
                         }
 
+                        if (size_t* property = std::get_if<size_t>(
+                                &pcieDevProperties["LanesInUse"]);
```

Comment:
I thought the D-Bus property was called something else?

Reply:
I am wrong this is LanesInUse
https://github.com/ibm-openbmc/phosphor-dbus-interfaces/commit/9fe8e065b4e73a888483bafbc02d8397883d3763